prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -16,11 +16,27 @@
 
   <div class="d-flex justify-content-between align-items-center mt-4">
     <span>
+      <app-alert *ngIf="enrollee?.possiblePaperEnrolmentMatch"
+                 type="warning">
+        <ng-container #alertTitle
+                      class="alert-title">
+          Multiple possible paper enrolment matches found
+        </ng-container>
+        <ng-container #alertContent
+                      class="alert-content">
+          Use the Enrollee search to determine which paper enrolments match
+        </ng-container>
+      </app-alert>
       <div *ngIf="enrolment?.enrollee.gpid"
-            class="mb-4 align-self-center">
+           class="mb-4 align-self-center">
           <strong>GPID:</strong> {{ enrolment?.enrollee.gpid }}
       </div>
-
+      <div *ngIf="enrollee?.linkedEnrolleeId">
+        <a [routerLink]=""
+           (click)="onRoute([enrollee?.linkedEnrolleeId, AdjudicationRoutes.ENROLLEE_OVERVIEW])">
+          {{ (enrolment?.enrollee.gpid | contains: paperEnrolleeGpidFilter) ? 'Linked' : 'Paper' }} Enrolment link
+        </a>
+      </div>
```

Comment:
It seems like we should never have a situation like this where the alert is shown and the link has also been established.  They seem to be exclusive of each other

![image](https://user-images.githubusercontent.com/7586665/142695573-1671ed3d-e1ff-4dcf-8c4f-fa827c8e8c80.png)

Reply:
There should never be a case where one enrollee is linked to more than one paper enrolment.

Reply:
Made them mutually exclusive